.github/workflows/ci.yml

```diff
@@ -14,16 +14,22 @@ defaults:
 
 env:
   CI_RELEASE: "${{ github.event_name == 'release' }}"
+  STACK_VERSION: "2.7.3"
 
 jobs:
   build:
     strategy:
       fail-fast: false # do not cancel builds for other OSes if one fails
       matrix:
-        # If upgrading Ubuntu, also upgrade it in the lint job below
-        os: [ "ubuntu-18.04", "macOS-10.15", "windows-2019" ]
+        include:
+          - # If upgrading the Haskell image, also upgrade it in the lint job below
+            os: "ubuntu-latest"
+            image: "haskell:8.10.7-stretch@sha256:100f8fb7d7d8d64adb5e106fe8136b8d4cbdc03aeb2cbd145a7597d74b69bafb"
```

Comment:
Yes. And instead of having a chance of mismatched GHC version in any release, I have tested that the build will indirectly "remind" for upgrading of the Haskell image with this error:
```
No compiler found, expected minor version match with ghc-8.10.7 (x86_64) (based on resolver setting in /__w/purescript/purescript/stack.yaml). 
427 To install the correct GHC into /root/.stack/programs/x86_64-linux/, try running "stack setup" or use the "--install-ghc" flag. To use your system GHC installation, run "stack config set system-ghc --global true", or use the "--system-ghc" flag.
```

Reply:
I have also added a note inside `stack.yaml` as a reminder for this.

Reply:
Great, thanks!